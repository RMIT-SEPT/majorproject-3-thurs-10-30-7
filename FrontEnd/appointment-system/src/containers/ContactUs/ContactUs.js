import React from 'react';
import Layout from '../Layout/Layout';

const ContactUs = (props) => {
  const redirectAbout = () => {
    props.history.push('/about');
  };
  return (
    <Layout>
      <div className='container'>
        <br />
        <h2 className='text-center'>Contact Us</h2>
        <hr />
        <br />
        <div className='d-flex justify-content-between'>
          <div className='d-flex flex-column'>
            <h4 className='text-center'>
              <i className='fas fa-phone'></i> Phone
            </h4>
            <p>
              Prefer to talk directly to someone? Contact us via the phone on XXXX-XXX-XXX.
            </p>
          </div>
          <div className='d-flex flex-column'>
            <div className='container'></div>
          </div>
          <div className='d-flex flex-column'>
            <h4 className='text-center'>
              <i className='fas fa-envelope'></i> Email
            </h4>
            <p>
              Or, email us at XXXX@XXXXXXXX.XXX for other enquiries.
            </p>
          </div>
        </div>
        <hr />
        <br />
        <div className='d-flex justify-content-center'>
          <button className='btn btn-link text-center' onClick={redirectAbout}>
            About Us
          </button>
        </div>
      </div>
    </Layout>
  );
};

export default ContactUs;
