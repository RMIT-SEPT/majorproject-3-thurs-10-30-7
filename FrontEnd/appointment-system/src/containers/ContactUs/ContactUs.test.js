import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ContactUs from "./ContactUs";

configure({ adapter: new Adapter() });

describe("<ContactUs/>", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ContactUs />);
  });

  it("should render contact us page", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
