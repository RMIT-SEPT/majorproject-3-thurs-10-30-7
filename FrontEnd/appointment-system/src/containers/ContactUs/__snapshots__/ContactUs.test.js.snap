// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContactUs/> should render contact us page 1`] = `ShallowWrapper {}`;
